packages/combobox/src/Combobox.tsx

```diff
@@ -754,15 +754,19 @@ export function Combobox<M extends boolean>({
       e: React.MouseEvent<HTMLButtonElement, MouseEvent>,
     ) => {
       if (!disabled) {
+        // Prevents triggering the setOpen function called by clicking anywhere within the input wrapper.
+        e.stopPropagation();
         updateSelection(null);
         onClear?.(e);
         onFilter?.('');
-        if (!isOpen) {
-          openMenu();
-        }
+        setInputFocus();
       }
     };
 
+    const handleClearButtonKeyDown = (e: React.KeyboardEvent) => {
+      e.stopPropagation();
+    };
```

Comment:
This still seems to be blocking Arrow key nav (in Multiselect story: make a selection, tab over to the clear button. Left arrow key should focus the input)

Reply:
The test that checks this is skipped for some reason... https://github.com/mongodb/leafygreen-ui/blob/brooke/fix-combobox/packages/combobox/src/Combobox.spec.tsx#L1102-L1111